src/FlaUILibrary/keywords/tree.py

```diff
@@ -0,0 +1,307 @@
+        | Argument   | Type   | Description                   |
+        | identifier | string | XPath identifier from element |
+        | msg        | string | Custom error message          |
+
+        Examples:
+        | ${Name}  Get All Visible Tree Items Name  <XPATH>  |
+        | Should Be Equal  ${Name}  <VALUE_TO_COMPARE> |
+
+        """
+        element = self._module.action(Element.Action.GET_ELEMENT, identifier, msg)
+        return self._module.action(Tree.Action.GET_VISIBLE_ITEMS_NAMES,
+                                   [self._module.cast_element_to_type(element, InterfaceType.TREE)],
+                                   msg)
+
+
+    @keyword
+    def expand_all_treeitems(self, identifier, msg=None):
+        """Expands every expandable Tree Items items of the given tree.
+
+        XPath syntax is explained in `XPath locator`.
+
+        If element could not be found by xpath an error message will be thrown.
+
+        Arguments:
+        | Argument   | Type            | Description                   |
+        | identifier | string          | XPath identifier from element |
+        | msg        | string | Custom error message                   |
+
+        Examples:
+        | Expand All Tree Items <XPATH>                 |
+        """
+        element = self._module.action(Element.Action.GET_ELEMENT, identifier, msg)
+        self._module.action(Tree.Action.EXPAND_ALL,
+                            [self._module.cast_element_to_type(element, InterfaceType.TREE)],
+                            msg)
+
+    @keyword
+    def collapse_all_treeitems(self, identifier, msg=None):
+        """Collapse every collapsable tree items of the given tree.
+
+        XPath syntax is explained in `XPath locator`.
+
+        If element could not be found by xpath an error message will be thrown.
+
+        Arguments:
+        | Argument   | Type            | Description                   |
+        | identifier | string          | XPath identifier from element |
+        | msg        | string | Custom error message                   |
+
+        Examples:
+        | Collapse All Tree Items <XPATH>                 |
+        """
+        element = self._module.action(Element.Action.GET_ELEMENT, identifier, msg)
+        self._module.action(Tree.Action.COLLAPSE_ALL,
+                            [self._module.cast_element_to_type(element, InterfaceType.TREE)],
+                            msg)
+
+    @keyword
+    def treeitem_should_be_visible(self, identifier, name, msg=None):
+        """Iterates every visible tree item. And fails if does not find the given name.
+
+        XPath syntax is explained in `XPath locator`.
+
+        If element could not be found by xpath an error message will be thrown.
+
+        Arguments:
+        | Argument   | Type            | Description                   |
+        | identifier | string          | XPath identifier from element |
+        | name       | string | Name of item                           |
```

Comment:
Tree name items. Documentation should be clear enough to describe index usage and name usage